assets/sass/components/global/_googlesitekit-data-block.scss

```diff
@@ -91,6 +91,17 @@
 			}
 		}
 
+		&--button-#{$button}.googlesitekit-data-block--is-gathering-data {
+
+			&::before {
```

Comment:
I had to diverge slightly from the IB here - please see **Relevant technical choices** in the PR description for a bit more on this.